src/components/Path/__tests__/Path.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+
+import { Path } from '../Path';
+import { PathStage } from '../PathStage';
+
+describe('<Path />', () => {
+  it('renders the correct html and classes', () => {
+    const mounted = render(
+      <Path button={(<p className="whatishappening">test</p>)}>
+        <PathStage label="testLabel" onStageClick={() => {}} />
+      </Path>
+    );
+    expect(mounted.find('div.slds-path-coach').length).toBe(1);
+    expect(mounted.find('div.slds-grid').length).toBe(1);
+    expect(mounted.find('div.slds-tabs_path').length).toBe(1);
+  });
+
+  it('renders the button', () => {
+    const mounted = render(
```

Comment:
you could extract this in a renderFunction like this:

`const getComponent = (props ={}) => shallow(...)`

and call it